prelude.rb

```diff
@@ -1,13 +1,13 @@
 class Binding
   # :nodoc:
-  def irb
+  def irb(*args)
```

Comment:
How about `(...)` instead?
```suggestion
  def irb(...)
```